content/en/docs/installation/upgrading/remove-deprecated-apis.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Removing Deprecated API Resources"
+linkTitle: "Removing Deprecated API Resources"
+weight: 20
+type: "docs"
+---
+
+We have deprecated the following cert-manager APIs: 
+- `cert-manager.io/v1alpha2`
+- `cert-manager.io/v1alpha3`
+- `cert-manager.io/v1beta1`
+- `acme.cert-manager.io/v1alpha2`
+- `acme.cert-manager.io/v1alpha3`
+- `acme.cert-manager.io/v1beta1`
+These APIs will be removed in cert-manager `v1.6.0`.
+If you have a cert-manager installation that is using or has previously used these deprecated APIs you may need to upgrade your cert-manager custom resources and CRDs. This needs to be done before upgrading to "cert-manager" `v1.6.0`. 
+
+
+## Upgrading existing cert-manager resources
+
+1. Familiarize yourself with the [official Kubernetes documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#writing-reading-and-updating-versioned-customresourcedefinition-objects) on CRD versioning.
+1. Make sure your installed cert-manager deployment is `v1.0.0` or later.
+2. Make sure any version-controlled cert-manager custom resource config files that refer to the deprecated APIs are updated to use the `cert-manager.io/v1` API and re-applied. This should update stored versions of the given resources.
+After that, follow the official Kubernetes documentation [here](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version):
```

Comment:
suggestion: tested locally: this doesn't render correctly without a space above it:

```suggestion

After that, follow the official Kubernetes documentation [here](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version):
 ```

Reply:
![render](https://user-images.githubusercontent.com/1972547/120334477-9270cf80-c2e8-11eb-9f45-73b263349097.png)